packages/react-table/src/components/Table/TableTypes.tsx

```diff
@@ -154,10 +155,10 @@ export interface IAction extends Omit<DropdownItemProps, 'title' | 'onClick'>, P
   itemKey?: string;
   /** Content to display in the actions menu item */
   title?: string | React.ReactNode;
-  /** Tooltip to display when hovered over the item */
-  tooltip?: React.ReactNode;
-  /** Additional props forwarded to the tooltip component */
-  tooltipProps?: any;
+  /** Render item as aria-disabled option */
+  isAriaDisabled?: boolean;
+  /** Props for adding a tooltip to a menu item. This is used to display tooltip when hovered over the item  */
+  tooltipProps?: MenuItemTooltipPropsObject;
```

Comment:
The `IAction` interface isn't on our props table in the surge because the `IActions` type isn't translating properly to the props table I believe. Might be out of scope for this PR.

Reply:
I removed `IActions` for now.  We can open a follow up issue.